### PR TITLE
Fix chrome login issue

### PR DESCRIPTION
**Pull request by SSshahan**
@akostadinov  @yapei  Make chromedriver to accept insecure cert automatically based on comment https://bugs.chromium.org/p/chromium/issues/detail?id=1056073#c6
@yanpzhan  @xiaocwan  FYI

### CODE REVIEW

**Comment by SSshahan** [comment]
pass log: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/73503/consoleFull

File: lib/webauto/web4cucumber.rb [hunk 1]
```diff
@@ -132,6 +132,9 @@ def browser
         end
       elsif @browser_type == :chrome
         logger.info "Launching Chrome"
+
+	#https://bugs.chromium.org/p/chromium/issues/detail?id=1056073
+	chrome_caps[:acceptInsecureCerts] = true
```
Comment (liangxia) [inline lib/webauto/web4cucumber.rb]: Bad indention ?

Comment (SSshahan) [reply lib/webauto/web4cucumber.rb]: Strange, well indention in my local editor. Updated the bad indention on web directly.

**Comment by akostadinov** [comment]
Thanks!

/lgtm

**Pull request merged**